contracts/dtoken/src/admin.rs

```diff
@@ -39,10 +39,10 @@ impl Contract {
     }
 
     pub fn set_total_reserves(&mut self, amount: Balance) -> Balance {
-        require!(
-            self.is_valid_admin_call(),
-            "This functionality is allowed to be called by admin or contract only"
-        );
+        // require!(
```

Comment:
@dariacherniak we don't need to leave commented code (as we have git to recover it)